mitodl/mit-open
# Rename env var prefix MITOPEN_ to MITOL_

### What are the relevant tickets?




Part of https://github.com/mitodl/hq/issues/5154

### Description (What does it do?)

Renames the environment variables to the more non product specific `MITOL_`.

Temporary code added to `manage.py` to use the legacy prefix if variables are present.

### How can this be tested?

- Checkout the first commit and verify you can `docker compose up` without errors 
- Checkout the second commit and verify the same
